src/main/java/teammates/storage/sqlapi/UsersDb.java

```diff
@@ -659,8 +669,5 @@ public Student updateStudent(Student student)
         if (getStudent(student.getId()) == null) {
             throw new EntityDoesNotExistException(ERROR_UPDATE_NON_EXISTENT);
         }
-
```

Comment:
Could you add back the whitespace here and for other files in the PR? So we can track code changes easier